server/src/controllers/client/updateClient.js

```diff
@@ -0,0 +1,66 @@
+const logger = require("pino")();
+const Client = require("../../models/client.model");
+
+const updateClientRequestHandler = async (req, res) => {
```

Comment:
Given the recent model changes we should update the date_updated attribute of the client model